modules/editing-model-version-metadata-in-a-model-registry.adoc

```diff
@@ -0,0 +1,40 @@
+:_module-type: PROCEDURE
+
+[id="editing-model-version-metadata-in-a-model-registry_{context}"]
+= Editing model version metadata in a model registry
+
+[role='_abstract']
+You can edit the metadata of model versions that are registered in {productname-short}, such as the version's description, labels, and custom properties.
+
+.Prerequisites
+* You have logged in to {productname-long}.
+ifndef::upstream[]
+* If you are using {productname-short} groups, you are part of the user group or admin group (for example, {oai-user-group} or {oai-admin-group}) in OpenShift.
+endif::[]
+ifdef::upstream[]
+* If you are using {productname-short} groups, you are part of the user group or admin group (for example, {odh-user-group} or {odh-admin-group}) in OpenShift.
+endif::[]
+* An available model registry exists in your deployment, and contains at least 1 registered model.
+* You have access to the model registry that contains the model version that you want to edit.
+
+.Procedure
+. From the {productname-short} dashboard, click *Model Registry*.
+. From the *Model registry* dropdown menu, select the model registry that contains the model version that you want to edit.
+. Click the name of a model to view more details.
++
+The details page for the model opens.
+. Click the name of a version to view more details.
++
+The details page for the version opens.
+. On the *Details* tab, you can edit the version metadata.
+.. In the *Description* section, click *Edit* to edit the description of the version.
+.. In the *Labels* section, click *Edit* to edit the labels of the version.
+.. In the *Properties* section, click *Add property* to add a new property to the version. 
+... To edit an existing property, click the action menu (*&#8942;*) beside the property, and then click *Edit*. 
+... To delete a property, click the action menu (*&#8942;*) beside the property, and then click *Delete*. 
+
+.Verification
+* You can view the updated metadata on the details page for the model version.
```

Comment:
[Suggestion] _You can view the updated metadata for the model version on the details page._

Reply:
I'm going to keep this one as is - it aligns with the wording we have for other procs and in DSP. Thank you!